collectors/python.d.plugin/am2320/README.md

```diff
@@ -0,0 +1,43 @@
+# _*_ coding: utf-8 _*_
+# Description: AM2320 netdata module
+# Author: tommybuck
+# SPDX-License-Identifier: GPL-3.0-or-Later
```

Comment:
I believe this header is supposed to go in the `am2320.chart.py` file.

Reply:
nice catch! 😄

Reply:
done